.ci/setup.sh

```diff
@@ -48,7 +48,7 @@ else  # Linux
     fi
 fi
 
-if [[ "${TASK:0:9}" != "r-package" ]]; then
```

Comment:
Same change here
https://github.com/microsoft/LightGBM/blob/415c0cb57794d184ea8639aa5a98a83610639976/.ci/test.sh#L11

Reply:
fixed in https://github.com/microsoft/LightGBM/pull/3726/commits/f886b142b272f099373f5cef7c03661c0c53a8a6